content/talks/agile-systems.rst

```diff
@@ -17,6 +17,14 @@ events:
     url: 'https://www.meetup.com/front-range-front-end/events/zwjmwlyxgbhb/'
     date: [2018, 4, 5]
     adr: 'Denver, CO'
+quotes:
+  - text: |
+      Every time I hear Miriam talk, I learn a ton and laugh a ton…
+      Watch this talk when it’s published!
```

Comment:
What does this second sentence mean? Isn't it already published?

Reply:
It wasn't when she tweeted that. We can remove it, but I thought it was still a nice sentiment.